<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="GameWidget" name="gameWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>900</height>
    </rect>
   </property>
  </widget>
  <widget class="GameOverWidget" name="gameOverWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="StartWidget" name="startWidget" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>9</y>
     <width>361</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <zorder>gameOverWidget</zorder>
  <zorder>gameWidget</zorder>
  <zorder>startWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StartWidget</class>
   <extends>QWidget</extends>
   <header>startwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameWidget</class>
   <extends>QWidget</extends>
   <header>gamewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameOverWidget</class>
   <extends>QWidget</extends>
   <header>gameoverwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
